### PR TITLE
Coverage & Clean Code:)

### PR DESCRIPTION
**Pull request by Askir**
Add tests for untested routes. And align the /index responses with the documentation in the wiki.
Reviews appreciated although this is not done yet!

### CODE REVIEW

File: spec/app.spec.js [hunk 1]
```diff
@@ -10,34 +10,145 @@ describe('app', () => {
...
-      request(app).post('/images')
-        .attach('image', 'spec/test.jpg')
-        .end((err, res) => {
-          expect(res.status).to.equal(200); // 'success' status
+      it('has status 200', (done) => {
+        expect(response.status).to.equal(200);
+        done();
+      });
+
+      it('warps image objects', (done) => {
```
Comment (arne-z) [inline spec/app.spec.js]:
```suggestion
      it('wraps image objects', (done) => {
```
Or was it really meant to be warp? In that case I don't quite understand.

File: src/controllers/ImageController.js [hunk 2]
```diff
@@ -4,13 +4,20 @@ import type { $Request as Request, $Response as Response } from 'express';
 import ImageService from '../services/ImageService';
 import logger from '../logger';
 
+function formatIndex(elijaResponse) {
+  return elijaResponse.results.map(image => ({
```
Comment (arne-z) [inline src/controllers/ImageController.js:9]: Not sure the ID is even needed anywhere outside of Esra, at the same time the score and the date (if it already exists in the database) might be interesting to return, so thresholds can be set, and responses ordered by score/date.

**Review by arne-z — CHANGES_REQUESTED** [review]

File: spec/app.spec.js (hunk 1 above)
Comment (Askir) [reply spec/app.spec.js]: wont fix, warp in the sense that it warps the json to something else.

File: src/controllers/ImageController.js (hunk 2 above)
Comment (Askir) [reply src/controllers/ImageController.js:9]: The image ID is needed to update the tags, routes to images use these uuids

**Review by arne-z — APPROVED** [review]

**Pull request merged**